spec/models/announcement_dismissal_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+RSpec.describe AnnouncementDismissal, type: :model do
```

Comment:
```diff
- Spec.describe AnnouncementDismissal, type: :model do
+ describe AnnouncementDismissal do
```

Reply:
Happy to change this but we should consider switching to the newer syntax: https://github.com/rspec/rspec-rails/commit/ca0d249858903949052e06884e8e7f9d596cdc79

Reply:
Good Q. Lets go with `describe` for now and make a ticket for https://github.com/rspec/rspec-rails/commit/ca0d249858903949052e06884e8e7f9d596cdc79 and we can just make one commit for the change if we decide to do it.

Reply:
Have changed this in 542ed06 & 04c27ba